### ISSUE TITLE
Release new SDK with configuration

### ISSUE THREAD

**Issue by ekrengel** [pulumi/pulumi-policy#176]
- Release SDK
- Add appropriate docs

Related to Issue #91

**Comment by ekrengel** [comment]
@justinvp I assigned this to you bc youve been doing the work.. not sure if you already have an open issue or not. If this is a duplicate, feel free to close. Also feel free to push to M33 if thats more realistic!

**Comment by justinvp** [comment]
Before we can publish a new version of `@pulumi/policy`, we'll need a new CLI and `@pulumi/pulumi` release.

**Comment by justinvp** [comment]
Going to close this as we've released `v0.5.0`. I'll open a new issue to track releasing `v1.0.0`.
